src/Autocomplete/src/Form/ParentEntityAutocompleteType.php

```diff
@@ -82,6 +82,8 @@ public function configureOptions(OptionsResolver $resolver)
             'security' => false,
             // set the max results number that a query on automatic endpoint return.
             'max_results' => 10,
+            /* @see AutocompleteEntityTypeSubscriber::removeBlacklistedOptions() */
```

Comment:
```suggestion
            /* @see AutocompleteEntityTypeSubscriber::withoutExcludedOptions() */
```

Reply:
Yeah I'm pushing that change now. Thanks!